backend-database-tech-it-easy.sql

```diff
@@ -0,0 +1,151 @@
+-------- CREATE TABLES --------
+
+CREATE TABLE Products (
+	id SERIAL PRIMARY KEY,
+	name VARCHAR(255) NOT NULL,
+	brand VARCHAR(255) NOT NULL,
+	price DECIMAL NOT NULL,
+	current_stock INT NOT NULL,
+	sold INT NOT NULL,
+	type VARCHAR(255) NOT NULL CHECK (type = 'television' OR type = 'remote control' OR type = 'ci module' OR type = 'wall bracket')
+);
+
+CREATE TABLE Items (
+	serial_number VARCHAR(255) NOT NULL PRIMARY KEY,
+	date_sold TIMESTAMP,
+	product_id INT NOT NULL REFERENCES Products(id)
+);
+
+CREATE TABLE Televisions (
+	id SERIAL PRIMARY KEY,
+	height INT NOT NULL,
+	width INT NOT NULL,
+	screen_quality VARCHAR(255) NOT NULL,
+	screen_type VARCHAR(255) NOT NULL,
+	wifi BIT NOT NULL,
+	smart_tv BIT NOT NULL,
+	voice_control BIT NOT NULL,
+	hdr BIT NOT NULL,
+	product_id INT NOT NULL REFERENCES Products(id)
+);
```

Comment:
In de televisie tabel zit de foreign keys voor de remote controller en CI module; Eerst moeten er dan nog 2 velden bij gemaakt worden voor de remote_id en ci-module-id onder je product_id vervolgens kan je met onderstaande de koppeling maken;
    CONSTRAINT fk_remotecontroller FOREIGN KEY (remote_id) REFERENCES remote_controllers(id),
   CONSTRAINT fk_cimodule FOREIGN KEY (ci_module_id) REFERENCES cimodules(id)